### DISCUSSION TITLE
[Prettier Plugin]

### DISCUSSION THREAD

**Discussion by AMR-21** [tailwindlabs/tailwindcss#12184] [Help]
Hello, the plugin does not work for me while using plain javascript objects, for example:
`{
 destructive:
          "bg-red-500  hover:bg-red-500/90 dark:bg-red-900 dark:text-gray-50 dark:hover:bg-red-900/90 text-yellow-500",
}`

text-yellow-500 should have been after bg-red , any suggestions or configurations?

**Comment by alex-krasikau** [comment]
You can try to utilize VS Code `tailwindCSS.experimental.classRegex` setting to define the pattern for your styles declaration.

This comment looks like the right solution for you - https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-917271069

```json
"tailwindCSS.experimental.classRegex": [["/\\*tw\\*/ ([^;]*);", "'([^']*)'"]]
```

```js
const fieldStyles = /*tw*/ {
    default: {
        label: 'py-2',
        input: 'px-3 py-2 font-medium text-sm',      
    },    
    inline: {
        label: 'py-2',
        input: 'px-3 py-2 font-medium text-sm',      
    },    
};
```
